proto/expr.proto

```diff
@@ -202,6 +203,7 @@ message ExprNode {
     data.Datum constant = 5;
     FunctionCall func_call = 6;
     UserDefinedFunction udf = 7;
+    bool now = 8;
```

Comment:
What's the semantic of `bool` here? I guess it's better to define an empty message `Now` for this.